specs/create_Edit_pool.adoc

```diff
@@ -0,0 +1,113 @@
+// vim: tw=79
+
+= Create and Edit Pool
+
+Write a functionality for creating and editing a Pool in Tendrl-Dashboard UI
+
+
+== Problem description
+
+For Creating the new Pool we need to Provide forms with different fields
+* Pool Name
+* Number of Pools to create
+* Type()
+* Replicas
+* Cluster it belongs
+* Journal configration
+* OSD's
+* PG Count
+* Owner
+* Quotas
```

Comment:
@shtripat Quotas total value is provided on the UI. So I think need it from the API.
You can see the 128GB written in UI. I am talking about that value. https://redhat.invisionapp.com/share/2T9LPMEEB#/screens/217733677